macos/Sources/Features/Command Palette/CommandPalette.swift

```diff
@@ -59,6 +59,7 @@ struct CommandOption: Identifiable, Hashable {
 
 struct CommandPaletteView: View {
     @Binding var isPresented: Bool
+    @Binding var isJumpPalette: Bool
```

Comment:
Remove this, doesn't have to be a binding, and just make it `placeholder: String` and let downstream callers set it.